### DISCUSSION TITLE
Oqtane 4.0.2 release

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3104] [Q&A]
@sbwalker & @leigh-pointer  any idea when the Oqtane 4.0.2 will be released?

Are aware that the upload functionality for both modules and theme is not working and the #3058 enhancement fixes this?

**Comment by sbwalker** [comment]
@vnetonline the problem with uploading theme and module packages was reported in #3048 - the upload is working however it mistakenly displays an error message in the UI. The issue was resolved in #3051. Based on the fact that the feature is not actually broken and there are multiple ways to work around it indicated that an immediate patch release was not necessary. 

I do not believe #3058 is related to package upload as it is focused on non-package folder support. 

4.0.2 will likely be released early next week.

**Reply by vnetonline** [reply]
Okay sound's good

**Comment by sbwalker** [comment]
@vnetonline 4.0.2 was just released

**Reply by vnetonline** [reply]
Okay that's excellent I will test
